examples/Post.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * This file is part of the php-state project.
+ *
+ * (c) Yannick Voyer <star.yvoyer@gmail.com> (http://github.com/yvoyer)
+ */
+
+namespace Star\Component\State\Example;
+
+use Star\Component\State\State;
+use Star\Component\State\StateContext;
+use Star\Component\State\StateMachine;
+
+class Post implements StateContext
+{
+    const DRAFT = 'draft';
```

Comment:
Add a documentation that transition from one state to many state will be automatically convert to one to one transition